corc-cascading/src/test/java/com/hotels/corc/cascading/OrcFileTest.java

```diff
@@ -728,6 +730,7 @@ public void writeViaTypeInfo() throws IOException {
     }
   }
 
+  @Ignore("Failing due to java.lang.IllegalArgumentException: Include vector the wrong length at org.apache.orc.impl.SchemaEvolution.<init>(SchemaEvolution.java:122)")
```

Comment:
As discussed with @teabot remove this method and the next one and also remove the data files they use since they're not compatible with the 2.x version of Hive.